### ISSUE TITLE
accessibility: Process list should announce labels for the values it reads

### ISSUE THREAD

**Issue by aleasto** [nokyan/resources#573]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

When using a screen reader, navigating through the list of applications/processes announces the process name followed by all of the values in the grid. However, these values are announced without any context so you wouldn't know what they represent.

### Describe the solution you'd like

Selecting a process should announce its name, then for every value it should announce "label: value".

### Describe alternatives you've considered

Instead of announcing all the values in sequence, perhaps you could have the right/left arrows highlight the individual columns and only announce the highlighted one, still with a label giving context to the value.

**Comment by nokyan** [comment]
Hi again, I'm currently working on this issue and I have a small question.
I made each value in the processes list be spoken as e. g. "Memory: 80.41 MB". The problem is, orca (at least in English) will pronounce the colon as "colon", so orca will say in this case "memory colon eighty dot four one M B" - should I keep it that in or should I remove the column?
